### PR TITLE
Adding Python Script For Magic Square

### PR DESCRIPTION
**Pull request by gulshanbaraik01**
Hello,

This is my PR #5 for Awesome Python Scripts. And #3rd one for Basic Scripts. Please review and add it.

Thanks,
Gulshan Baraik

### CODE REVIEW

File: Basic-Scripts/magic_square1.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Example:
+    2 7 6 
+    9 5 1 
+    4 3 8 
+    The sum of each row/column/diagonal is: 15.0
+    
+'''
+
+# Function to create Magic Square
+def magic_square(n):
```
Comment (SKAUL05) [inline Basic-Scripts/magic_square1.py]: Change file_name to magic_square
Include docstrings for function documentation.
Use meaningful variable names

File: Basic-Scripts/magic_square1.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    for i in range(n):
+        for j in range(n):
+            print(magicSquare[i][j], end=" ")
+        print()
+        
+    print("The sum of each row/column/diagonal is: "+str(n*(n**2+1)/2))
+
+# Main Function for Magic Square        
+if __name__ == '__main__':
+    magic_square(3)
```
Comment (SKAUL05) [inline Basic-Scripts/magic_square1.py]: Take Input from user
Provide Sample I/O

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Comment by gulshanbaraik01** [comment]
Hello, I updated as per your recent suggestions. Please add it, Thanks!!.

**Review by SKAUL05 — APPROVED** [review]
@ankitdobhal Review & Merge

**Review by powerexploit — APPROVED** [review]

**Pull request merged**